go/datas/remote_database_handlers.go

```diff
@@ -380,14 +380,14 @@ func handleRootPost(w http.ResponseWriter, req *http.Request, ps URLParams, cs c
 		d.Panic("Can't set Root to a non-present Chunk")
 	}
 
-	// Ensure that proposed new Root is a Map and, if it has anything in it, that it's <String, <Ref<Commit>>
+	// Ensure that proposed new Root is a Map and, if it has anything in it, that it's <String, <Ref<Value>>
 	v := types.DecodeValue(c, nil)
 	if v.Type().Kind() != types.MapKind {
 		d.Panic("Root of a Database must be a Map")
 	}
 	m := v.(types.Map)
-	if !m.Empty() && !isMapOfStringToRefOfCommit(m) {
-		d.Panic("Root of a Database must be a Map<String, Ref<Commit>>, not %s", m.Type().Describe())
+	if !m.Empty() && !isMapOfStringToRefOfValue(m) {
```

Comment:
This is a bummer and kind of gets at the question of what invariants are we trying to enforce. To be precise, this no longer offers assurance that the entire database hasn't be corrupted by setting the root to be a `Map<String, Ref<Garbage>>`.

If want to retain the same level of integrity, I think the obvious thing to do is to ALSO diff the old map to the new map and assert that all *new* map `Ref<Values>` are  actually`Ref<Commit>`.

@aboodman Thoughts?